core/src/main/java/org/openstack4j/openstack/networking/domain/NeutronRouterConntrackHelper.java

```diff
@@ -0,0 +1,225 @@
+package org.openstack4j.openstack.networking.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.google.common.base.MoreObjects;
+import org.openstack4j.model.network.RouterConntrackHelper;
+import org.openstack4j.model.network.builder.RouterConntrackHelperBuilder;
+import org.openstack4j.openstack.common.ListResult;
+
+import java.util.List;
+
+/**
+ * Single entity for Routers Conntrack Helper (CT) target rules
+ *
+ * @author bboyHan
+ */
+@JsonRootName("conntrack_helper")
+public class NeutronRouterConntrackHelper implements RouterConntrackHelper {
+
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("router_id")
+    private String routerId;
+    @JsonProperty("helper")
+    private String helper;
+    @JsonProperty("protocol")
+    private String protocol;
+    @JsonProperty("port")
+    private Integer port;
+
+    public static RouterConntrackHelperBuilder builder() {
+        return new RouterConntrackHelperConcreteBuilder();
+    }
+
+    @Override
+    public RouterConntrackHelperBuilder toBuilder() {
+        return new RouterConntrackHelperConcreteBuilder(this);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void setRouterId(String routerId) {
+        this.routerId = routerId;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getRouterId() {
+        return routerId;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getHelper() {
+        return helper;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getProtocol() {
+        return protocol;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Integer getPort() {
+        return port;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String toString() {
+        return MoreObjects.toStringHelper(this).omitNullValues()
+                .add("routerId", routerId).add("helper", helper)
+                .add("protocol", protocol).add("port", port)
+                .toString();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+		@Override
```

Comment:
Please respect surrounding indentation or reformat the code once your changes are done. We have https://github.com/openstack4j/openstack4j/blob/master/.editorconfig now...